notebooks/ids_detection/IDS_using_LODA.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>function names are great and clear, still, I would add the explanation below before these definitions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Fixed.